### PR TITLE
Use "not" rather than == false when checking for existing dhparams file

### PR DESCRIPTION
**Pull request by robbyoconnor**
See [the ansible docs on conditions](http://docs.ansible.com/ansible/playbooks_conditionals.html#the-when-statement) -- I've always hated seeing people comparing booleans in this manner...

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   shell: /usr/bin/openssl dhparam -out "{{ ssl_dh_file }}" {{ ssl_dh_size }}
   args:
     chdir:
-  when: (encryption != "off") and (_dh_file_st.stat.exists == false)
+  when: (encryption != "off") and (not _dh_file_st.stat.exists)
```
Comment (gronke) [inline tasks/main.yml]: Good catch! For better readability we could even remove the second parentheses.

Comment (robbyoconnor) [reply tasks/main.yml]: I'll do that

**Comment by robbyoconnor** [comment]
Fixed.

**Comment by robbyoconnor** [comment]
Should be good to merge @gronke

**Pull request merged**